src/types.rs

```diff
@@ -49,9 +49,19 @@ pub struct HWIAddress {
 
 #[derive(Clone, Eq, PartialEq, Debug, Deserialize)]
 pub struct HWIPartiallySignedTransaction {
+    #[serde(deserialize_with = "deserialize_psbt")]
```

Comment:
Why is this needed? PSBT already has the deserialize method https://docs.rs/bitcoin/latest/bitcoin/util/psbt/struct.PartiallySignedTransaction.html#impl-Deserialize%3C%27de%3E

Reply:
I agree that it shouldn't be needed, but it doesn't work without it. Serde complains that `expected struct PartiallySignedTransaction while deserializing`.

Reply:
Mh, weird. Ok to leave it like that then